hyukju/app.js

```diff
@@ -41,6 +41,134 @@ app.post("/users", async (req, res, next) => {
   res.status(201).json({ message: "successfully created" });
 });
 
+app.post("/posts", async (req, res, next) => {
+  const { title, content, user_id } = req.body;
```

Comment:
```suggestion
  const { title, content, userId } = req.body;
```
javascript에서는 snake_case가 아닌 camelCase를 쓰는것이 컨벤션 입니다!